testing/agent-test-extension/src/main/java/com/splunk/opentelemetry/testing/IgnoredTestTypesConfigurer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.testing;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.instrumentation.api.config.Config;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesBuilder;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesConfigurer;
+
+@AutoService(IgnoredTypesConfigurer.class)
+public class IgnoredTestTypesConfigurer implements IgnoredTypesConfigurer {
+  @Override
+  public void configure(Config config, IgnoredTypesBuilder builder) {
+    // we don't want to instrument auto-generated mocks
+    builder.ignoreClass("org.mockito");
```

Comment:
I've double-checked all my latest changes in the upstream, mockito was NEVER ignored... I have no idea how this worked up till now 🤷